chrononaut/change_info.py

```diff
@@ -107,8 +102,14 @@ def append_recorded_changes(obj, session):
         )
         raise ChrononautException(msg)
 
-    obj.change_info = fetch_change_info(obj)
-    obj.changed = datetime.now(pytz.utc)
+    # backwards compatibility workaround: generate a {'user_id', 'remote_addr', 'extra'} structure
```

Comment:
Do we need that?

Reply:
The `RecordChanges` mixin does get used e.g. in mainline code. This change allows for more backwards compatibility with existing code, but we could just change it since we're doing a major update anyway 🤷 
What do you think?